### PR TITLE
:recycle: Refactor all patients

### PR DESCRIPTION
**Pull request by issa-diallo**

### CODE REVIEW

File: src/pages/PatientsPage.tsx [hunk 1]
```diff
@@ -11,18 +11,23 @@ const PatientsPage = (): JSX.Element => {
 
   return (
     <>
-      <div style={{ display: 'flex', alignItems: 'center' }}>
-        <h1>{t('title.myPatients')}</h1>
-        <Button variant='primary' className='ms-auto' href='/patients/create'>
+      <div style={{ position: 'fixed', bottom: '100px', right: '20px', zIndex: '1' }}>
```
Comment (AndreMiras) [inline src/pages/PatientsPage.tsx:14]: :eyes:

File: src/pages/PatientsPage.tsx [hunk 2]
```diff
@@ -11,18 +11,23 @@ const PatientsPage = (): JSX.Element => {
...
       </div>
-      <Container>
-        {patients.map((patient) => (
-          <PatientLine key={patient.id} patient={patient} />
-        ))}
-      </Container>
+      <div className='d-flex align-items-center justify-content-center bg-info rounded-bottom py-4'>
+        <h1>{t('title.myPatients')}</h1>
+      </div>
+      <div style={{ marginBottom: '100px' }}>
```
Comment (AndreMiras) [inline src/pages/PatientsPage.tsx:22]: :eyes:

File: src/pages/PatientsPage.tsx (hunk 1 above)
Comment (issa-diallo) [reply src/pages/PatientsPage.tsx:14]: <div className="position-fixed bottom-100 end-20 z-index-1">

File: src/pages/PatientsPage.tsx (hunk 2 above)
Comment (issa-diallo) [reply src/pages/PatientsPage.tsx:22]: `<div className="mb-100">`

File: src/pages/PatientsPage.tsx (hunk 1 above)
Comment (issa-diallo) [reply src/pages/PatientsPage.tsx]: Resolved with `      <div className='position-fixed bottom-0 end-0 me-3 mb-5 pb-5' style={{ zIndex: '1030' }}>`

File: src/pages/PatientsPage.tsx (hunk 2 above)
Comment (issa-diallo) [reply src/pages/PatientsPage.tsx]: Resolved with `      <div className='mb-5 pb-5'>
`

**Review by AndreMiras — APPROVED** [review]

File: src/pages/PatientsPage.tsx [hunk 3]
```diff
@@ -11,18 +11,23 @@ const PatientsPage = (): JSX.Element => {
 
   return (
     <>
-      <div style={{ display: 'flex', alignItems: 'center' }}>
-        <h1>{t('title.myPatients')}</h1>
-        <Button variant='primary' className='ms-auto' href='/patients/create'>
+      <div className='position-fixed bottom-0 end-0 me-3 mb-5 pb-5' style={{ zIndex: '1030' }}>
+        <Button variant='info' href='/patients/create'>
```
Comment (AndreMiras) [inline src/pages/PatientsPage.tsx:15]: This button deserves its own component since it's used somewhere else in the preview PR

**Review by AndreMiras — APPROVED** [review]

**Pull request merged**